src/suggestion_engine/server/index.js

```diff
@@ -4,6 +4,21 @@ export async function getSuggestions([mode, searchString]) {
   return providers[mode](searchString);
 }
 
+async function focusOrCreateTab(url) {
+  const matchingTabs = await browser.tabs.query({ url });
+
+  if (matchingTabs && matchingTabs.length > 0) {
+    // If multiple matching tabs then just focus the first one
+    const existingTab = matchingTabs[0];
+
+    await browser.tabs.update(existingTab.id, { active: true });
+    await browser.windows.update(existingTab.windowId, { focused: true });
+    return;
+  }
```

Comment:
Small thing but I would prefer getting rid of the return and placing the await statement in an else block here.